__tests__/app.test.js

```diff
@@ -35,29 +35,78 @@ describe("API Topics", () => {
         .get("/api/topics")
         .expect(200)
         .then((response) => {
-            response.body.topics.forEach((topic) => {
-                expect(typeof topic.slug).toBe("string");
-                expect(typeof topic.description).toBe("string");
-            })
+          response.body.topics.forEach((topic) => {
+            expect(typeof topic.slug).toBe("string");
+            expect(typeof topic.description).toBe("string");
+          });
         });
-    })
+    });
     it("should return 3 topics", () => {
       return request(app)
-      .get('/api/topics')
-      .expect(200)
-      .then((response) => {
-        expect(response.body.topics.length).toBe(3)
-      })
-    })
+        .get("/api/topics")
+        .expect(200)
+        .then((response) => {
+          expect(response.body.topics.length).toBe(3);
+        });
+    });
     it("should return the right values, checking first topic", () => {
       return request(app)
-      .get('/api/topics')
-      .expect(200)
-      .then((response) => {
-        const firstResponse = response.body.topics[0]
-        expect(firstResponse.description).toBe('The man, the Mitch, the legend')
-        expect(firstResponse.slug).toBe('mitch')
-      })
+        .get("/api/topics")
+        .expect(200)
+        .then((response) => {
+          const firstResponse = response.body.topics[0];
+          expect(firstResponse.description).toBe(
+            "The man, the Mitch, the legend"
+          );
+          expect(firstResponse.slug).toBe("mitch");
+        });
+    });
+  });
+});
+
+describe("API Articles", () => {
+  describe("200 GET /api/articles/:article_id", () => {
+    it("200 Success - should return articles with the correct key value pairs", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((response) => {
+          response.body.article.forEach((story) => {
+            expect(story).toHaveProperty("article_id");
+            expect(story).toHaveProperty("title");
+            expect(story).toHaveProperty("topic");
+            expect(story).toHaveProperty("author");
+            expect(story).toHaveProperty("body");
+            expect(story).toHaveProperty("created_at");
+            expect(story).toHaveProperty("votes");
+            expect(story).toHaveProperty("article_img_url");
+
+            expect(typeof story.title).toBe("string");
+            expect(typeof story.topic).toBe("string");
+            expect(typeof story.author).toBe("string");
+            expect(typeof story.body).toBe("string");
+            expect(typeof story.created_at).toBe("string");
+            expect(typeof story.votes).toBe("number");
+            expect(typeof story.article_img_url).toBe("string");
+            expect(story.article_img_url.startsWith("http")).toBe(true);
+          });
+        });
```

Comment:
I would be more specific here, you can do a toEqual on the whole article and check that it looks like you expect. At the minimum I would check that the article id is 1 for example so you know you have the correct article. If you return rows[0] in the model you won't need a forEach either, just return the object rather than the whole array